config/metadata/local_info_metadata.yml

```
@@ -2,3 +2,4 @@ repository_tesim: 'Repository'
 local_identifier_ssm: 'Local identifier'
 oclc_ssi: 'OCLC Number'
 ark_ssi: 'ARK'
+opac_url_tesim: 'Opac url'
```

```suggestion
opac_url_ssi: 'Opac url'
```
Does this field go in more than one iml file

---

Does this field go in more than one yml file.